stackslib/src/main.rs

```diff
@@ -1470,6 +1470,35 @@ simulating a miner.
         process::exit(0);
     }
 
+    if argv[1] == "replay-naka-block" {
+        let chain_config =
+            if let Some(network_flag_ix) = argv.iter().position(|arg| arg == "--network") {
+                let Some(network_choice) = argv.get(network_flag_ix + 1) else {
+                    eprintln!("Must supply network choice after `--network` option");
+                    process::exit(1);
+                };
+
+                let network_config = match network_choice.as_str() {
```

Comment:
This should probably be case insensitive no?

Reply:
Sure -- updated in https://github.com/stacks-network/stacks-core/pull/5346/commits/84950edfa160e6662b3cbd41a14522fcf2fb9ba7